examples/README-authz.md

```diff
@@ -0,0 +1,403 @@
+
+
+## Building the Example Project
+
+Before using the example, we first need to build the project, and prepare resources.
+
+First change the current directory to `examples/docker`:
+
+    cd examples/docker
+
+Now build the project, and prepare resources:
+
+    mvn clean install -f ../..
+    mvn clean install
+
+We are now ready to start up the containers and see Keycloak Authorization Services in action.
+
+
+## Starting Up the Containers
+
+First make sure any existing containers with the same name are removed, otherwise we might use previous configurations:
+
+    docker rm keycloak kafka zookeeper
+    
+Let's start up all the containers with authorization configured, and we'll then perform any manual step, and explain how everything works.
+
+    docker-compose -f compose.yml -f keycloak/compose.yml -f keycloak-import/compose.yml \
+      -f kafka-oauth-strimzi/compose-authz.yml up --build
+
+When everything starts up without errors we should have one instance of `keycloak` listening on localhost:8080.
+
+
+## Using Keycloak Admin Console to Configure Authorization
+ 
+You can login to the Admin Console by opening `http://localhost:8080/auth/admin` and using `admin` as both username, and a password.
+
+In the upper left corner under Keycloak icon you should see `Master` selected as a current realm.
+Moving the mouse pointer over it should reveal two additional realms - `Demo` and `Kafka-authz`.
+
+For this example we are interested in the `kafka-authz` realm.
+Selecting it will open the `Realm Settings` for the `kafka-authz` realm.
+Next to `Realm Settings` there are other sections we are interested in - `Groups`, `Roles`, `Clients` and `Users`.
+
+Under `Groups` we can see several groups that can be used to mark users as having some permissions.
+In Keycloak the groups can be stored in an LDAP identity provider.
+That makes it possible to make some user a member of some group - through custom LDAP server admin UI for example, which grants them some permissions on Kafka resources.
+
+Under `Users` click on `View all users` button to see two users defined - `alice` and `bob`. They are both members of some group. 
+In Keycloak the users can be stored in an LDAP identity provider.
+
+Under `Roles` we can see several realm roles which can be used to mark users or clients as having some permissions.
+Roles are a concept analogous to groups. One exception in Keycloak is that roles can not be stored in an LDAP identity provider.
+
+Under `Clients` we can see some additional clients configured - `kafka`, `kafka-cli`, `team-a-client`, `team-b-client`.
+The client with client id `kafka` is used by Kafka Brokers to perform the necessary OAuth2 communication for access token validation, 
+and to authenticate to other Kafka Broker instances using OAuth2 client authentication.
+This client also contains Authorization Services resource definitions, policies and authorization scopes used to perform authorization on Kafka Brokers.
```

Comment:
To many users, this does not mean anything. Can we get some clarity on what these are here?

Reply:
These are somewhat abstract concepts, not entirely easy to explain. For starters we could add a link to [Keycloak Authorization Services docs](https://www.keycloak.org/docs/latest/authorization_services/index.html).
Maybe manage to put together a short explanation that would be enough to get started.